samples/cpp/image_generation/README.md

```diff
@@ -148,3 +149,35 @@ The resuling image is:
    ![](./inpainting.bmp)
 
 Note, that LoRA, heterogeneous execution and other features of `Text2ImagePipeline` are applicable for `InpaintingPipeline`.
+
+## benchmarking sample for text to image pipeline
+
+This `benchmark_text2image.cpp` sample script demonstrates how to benchmark the text to image pipeline. The script includes functionality for warm-up iterations, generating image, and calculating various performance metrics.
```

Comment:
can we generalize this benchmark to support inpainting / image to image as well?
We can add an argument to specify pipeline type